spinnman/messages/scp/impl/scp_utils.py

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
To both this and the above comment, its all about speed.  If you add the checks, you can't get the speeds.  If you use functions over constants, you can't get the speed. Welcome to python ;)